plugins/receiver/grpc/common.go

```diff
@@ -0,0 +1,29 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package grpc
+
+import (
+	"google.golang.org/grpc"
+
+	"github.com/apache/skywalking-satellite/protocol/gen-codes/satellite/protocol"
+)
+
+type CommonGRPCReceiverFields struct {
+	Server        *grpc.Server
+	OutputChannel chan *protocol.Event // The channel is to bridge the LogReportService and the Gatherer to delivery the data.
```

Comment:
As you moved the `OutputChannel` and `Server` into the CommonGRPCReceiverFields, should you consider to move the initialization into here too?